docs/1_getting_started.md

```diff
@@ -54,7 +54,7 @@ from fundus import PublisherCollection, Crawler
 # initialize the crawler for news publishers based in the US
 crawler = Crawler(PublisherCollection.us)
 
-# crawl 2 articles and print
+# crawl one article from each publisher based in the us and print it
```

Comment:
That's not what `max_articles` does. I would change line 49 `Let's crawl one news article ... ` -> `Let's crawl a random news article ...` and the comment to `crawl one random article and print it`